<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Truco Paulista</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCarta1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Carta 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCarta2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Carta 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCarta3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Carta 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTruco">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Pedir Truco</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblManilha">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Manilha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPontos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Pontos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTruco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rodada valendo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMesa">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>440</width>
      <height>130</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignTop</set>
    </property>
    <property name="text">
     <string>Cartas na mesa:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
